docs/Dependencies.md

```diff
@@ -13,6 +14,7 @@
   * gtest
   * opencl
   * libuuid
+  * mosquitto
```

Comment:
should maybe get marked as (optional) - as it's only needed to compile the Traze Client (https://github.com/iteratec/traze)

Reply:
Removed the package install and marked it as optional.